crates/ty_ide/src/symbols.rs

```diff
@@ -282,6 +283,27 @@ impl SymbolKind {
             SymbolKind::Import => "Import",
         }
     }
+
+    /// Maps this to a "completion" kind if a sensible mapping exists.
+    pub fn to_completion_kind(self) -> Option<CompletionKind> {
+        Some(match self {
+            SymbolKind::Module => CompletionKind::Module,
+            SymbolKind::Class => CompletionKind::Class,
+            SymbolKind::Method => CompletionKind::Method,
+            SymbolKind::Function => CompletionKind::Function,
+            SymbolKind::Variable => CompletionKind::Variable,
+            SymbolKind::Constant => CompletionKind::Constant,
+            SymbolKind::Property => CompletionKind::Property,
+            SymbolKind::Field => CompletionKind::Field,
+            SymbolKind::Constructor => CompletionKind::Constructor,
+            SymbolKind::Parameter => CompletionKind::Variable,
+            SymbolKind::TypeParameter => CompletionKind::TypeParameter,
+            // Not quite sure what to do with this one. I guess
+            // in theory the import should be "resolved" to its
+            // underlying kind, but that seems expensive.
+            SymbolKind::Import => return None,
```

Comment:
Yeah, I think it's fine to not do this now. Pylance does provide the icon and the documentation as well.